azext_iot/tests/test_iot_central_int.py

```diff
@@ -190,7 +190,7 @@ def test_central_device_registration_info_filter_unassociated(self):
             device_info_results.append(device.get("device_info"))
 
         for device in device_info_results:
-            assert device.get("deviceStatus") == device_status_expected
+            assert device.get("device_status") == device_status_expected
```

Comment:
I'd suggest adding coverage for restriction of what you should expect at the console. I.e. write a test such that a dev can't add unexpected properties to the payload without the test failing.